core/src/main/res/values-qq/strings.xml

```diff
@@ -31,7 +31,7 @@
   <string name="no_thanks">{{Identical|No thanks}}</string>
   <string name="rate_dialog_neutral">{{Identical|Later}}</string>
   <string name="open">{{Identical|Open}}</string>
-  <string name="library">{{Identical|Library}}</string>
+  <string name="library">Displayed in application menu</string>
```

Comment:
@Abijeet I think some wires got crossed here with the quite major change I made to the string files, consolidating duplicates/ removing unused strings. It seems "Library" has been translated in a few languages to mean "Get Content"

Reply:
@macgills - Hmm, I'm not exactly sure what happened here. This is now fixed and the source language shows the correct string. 

I think there was a problem with rename matching on our end. I recommend to not merge this patch for sometime until translators have fixed these incorrect translation.

Do you see any other bad translations or just this one?

Reply:
I think it is all just related to `library` for the moment, `complete` has been altered in a few places but it still looks correct to my untrained eye

Reply:
The next update should fix the incorrect translations for "Library". That will be pushed out next Monday.

Reply:
I think this can be merged now after auto-fixing the following issue via Android studio,
```
 /home/runner/work/***-android/***-android/core/src/main/res/values-ar/strings.xml:148: Error: Replace straight quotes (") with directional quotes (“”, &#8220; and &#8221;) ? [TypographyQuotes]
```